tests/FakeItEasy.Specs/ConfigurationSpecs.cs

```diff
@@ -786,6 +787,22 @@ public static void DoesNothingOnTaskReturningMethod(IFoo fake, Task result)
                 .x(() => result.Status.Should().Be(TaskStatus.RanToCompletion));
         }
 
+        [Scenario]
+        public static void CallingAnUnconfiguredVoidMethodWhenVoidFakeOptionsBuilderExists(IFoo fake, Exception? exception)
+        {
+            "Given a fake options builder that can build options for fake of type void"
+                .See<VoidOptionsBuilder>();
+
+            "And a fake of a type with a void method"
+                .x(() => fake = A.Fake<IFoo>());
+
+            "And the void method is called on the fake"
+                .x(() => exception = Record.Exception(() => fake.Bar()));
+
+            "Then it does not throw"

```

Comment:
I nearly asked why we don't just check the return value. :wink:

Reply:
Yeah, it should be an instance of void!